dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/FileFormatHelper.java

```diff
@@ -66,4 +75,18 @@ public static void updateFileFormats(Set<? extends Version> versions, final File
         });
         return fileFormatsFromDB;
     }
+
+    public static String calcSHA1(String content) {
+        //return org.apache.commons.codec.digest.DigestUtils.sha1Hex(content);
+        String sha1 = null;
+        try {
+            MessageDigest messageDigest = MessageDigest.getInstance("SHA-1");
+            messageDigest.update(content.getBytes("UTF-8"), 0, content.length());
```

Comment:
Can content ever be null? I suspect not, but you might want to make sure in the database and/or add a guard.

Reply:
added if checking if content is null